packages/editor-ui/src/components/ParameterInput.vue

```diff
@@ -959,6 +959,12 @@ export default mixins(
 						this.valueChanged(typeof value !== 'undefined' ? value : null);
 					}
 				} else if (command === 'refreshOptions') {
+					if (this.isResourceLocatorParameter) {
+						const resourceLocator = this.$refs.resourceLocator;
+						if (resourceLocator) {
+							(resourceLocator as Vue).$emit('refreshList');
```

Comment:
not sure.. one approach I could think of is creating an event bus and passing it down but that's not much better.. or moving data to a store.. but not sure if a store makes sense here... since we don't reuse the data and only needed when directly using a parameter..